### ISSUE TITLE
[Due for payment 2026-03-30] [$250] [Due for payment 2026-01-30] Screen Reader: Track Distance: The button 'x' is announced non-descriptively

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76952]
## Action Performed:
Prerequisites:
- The user is signed in
- The start and stop points have been selected on the track distance page

1. Using Chrome + JAWS, open the URL - https://new.expensify.com/
2. Navigate to the "+" icon and press Enter
3. Tab to the "Track distance" menu item and press Enter to open the Pop-up
4. After selecting the "Start/stop" points, Tab to the "Next" button and press Enter
5. Tab to the "Expensify.org / Teachers Unite" and press Enter to open
6. Tab to the close "X" button
7. Observe the screen reader announcement

## Expected Result:
The screen reader should clearly and descriptively announce the purpose of the button, e.g.:
- 'dismiss 'submit an expense banner' or similar

## Actual Result:
The button 'x' is announced non-descriptively:
- 'close'
The screen reader user will not understand what will be closed.

HTML:
<button aria-label="Close" role="button" data-selection-scrapper-hidden-element="true" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-1awozwy r-eu3ka r-1777fci r-1aockid r-9ffhgg r-lrvibr" type="button" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="false" pressed="false" fill="#8B9C8F"><path d="M14.6 17.4c.8.8 2 .8 2.8 0s.8-2 0-2.8L12.8 10l4.6-4.6c.8-.8.8-2 0-2.8s-2-.8-2.8 0L10 7.2 5.4 2.6c-.8-.8-2-.8-2.8 0s-.8 2 0 2.8L7.2 10l-4.6 4.6c-.8.8-.8 2 0 2.8s2 .8 2.8 0l4.6-4.6z"></path></svg></div></button>

## Area issue was found in:
Track Distance

## Failed WCAG checkpoints
WCAG 2.4.6 Headings and Labels
https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels

## User impact:
Descriptive headings are necessary for users who have disabilities that make reading slow and for people with limited short-term memory. These users benefit when section titles make it possible to predict what each section contains. Form input controls with labels that clearly describe the content that is expected to be entered helps users know how to successfully complete the form.

## Suggested resolution:
Update the aria-label to announce the button, e.g.:
- 'dismiss 'submit an expense banner' or similar
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-label

## Environment(s)
Windows Windows 11 Home - Chrome

## Screenshots/Videos
<img width="1919" height="962" alt="Image" src="https://github.com/user-attachments/assets/f9105add-400e-4ee5-bc1d-e4e77d779a8e" />

https://github.com/user-attachments/assets/b5548f4f-d52d-4d42-bea1-ac0c2cde53db

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6997322
**Issue reported by:** NA
**Slack conversation:** NA

<details><summary>Issue Owner</summary>Current Issue Owner: @mallenexpensify</details>
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~022030838426724567825</li>
        <li>Upwork Job ID: 2030838426724567825</li>
        <li>Last Price Increase: 2026-03-09</li>
<li>Automatic offers: </li>
<ul>
<li>FitseTLT | Contributor | 110741646</li>
</ul></li>
    </ul>
</details>
